# bombard_termianl_nav_node
target_lat: 23.1065565
target_long: 120.2105330
target_alt: 30
target_pre_wp: 7
target_now_wp: 8 
